lightning_pose/train.py

```diff
@@ -181,17 +204,14 @@ def train(cfg: DictConfig) -> None:
         preds_file=preds_file,
     )
     # compute and save various metrics
-    try:
-        # take care of multiview case, where multiple csv files have been saved
-        preds_files = [
-            os.path.join(hydra_output_directory, path) for path in
-            os.listdir(hydra_output_directory) if path.endswith(".csv")
```

Comment:
Reason for changing this was that path.endswith(".csv") was picking up the saved labeled csv files.